### ISSUE TITLE
Depth of 2D images

### ISSUE THREAD

**Issue by MeghaMaheshwari** [mrharicot/monodepth#23]
Is it possible to use the pre-trained weights from kitti model and use them to run my own test set of 2D images to get the depth estimation. How can I achieve this on my data where I dont have both left and right images.

**Comment by mrharicot** [comment]
Hi,

Yes you can!
The simplest way right now is to list the path to your images into a text file (one by line) and to use this as your --filenames_file
